### ISSUE TITLE
[Feature Request] Auto Demo Recording

### ISSUE THREAD

**Issue by dumptruckDS** [andrei-drexler/ironwail#265]
Mark V has an autodemo feature that's quite handy. I am aware of the sentiment that demos aren't as useful in the age of OBS, Twitch, YouTube and Shadowplay. However, it would be a great addition to IW if it's not too hard to implement. I'd love to see it organized as you have with the screenshots folder and those naming schemes. It would be great to "set and forget" this cvar and have the ability to start a new recording automatically after a player's death. I don't recall at the moment if that's how Mark V works.

**Comment by 4LT** [comment]
Yeah, this would be great for those moments you weren't planning on recording, but wish you had

**Comment by Konkonfaux** [comment]
It would help with playtesting too, if you could die, reload, and that would automatically start a new demo.
